scripts/core/ui/components/SelectUser.tsx

```diff
@@ -1,120 +1,139 @@
+                                    {
+                                        username: {
+                                            $regex: searchString,
+                                            $options: '-i',
+                                        },
+                                    },
+                                ],
+                            }
+                            : {},
+                    );
+
+                    return new Promise((resolve) => {
+                        fetch(`${appConfig.server.url}/users?where=${query}&max_results=50`, {
+                            signal: this.abortController.signal,
+                            method: 'GET',
+                            headers: {
+                                'Authorization': ng.get('session').token,
+                            },
+                            mode: 'cors',
+                        })
+                            .then((res) => res.json())
+                            .then((res) => {
+                                resolve(res._items);
+                            })
+                            .catch((err) => {
+                                if (err?.name !== 'AbortError') {
+                                    throw err;
+                                }
+                            });
+                    });
                 }}
-                renderItem={(user) => (
-                    <ListItem fullWidth noBackground noShadow>
-                        <ListItemColumn noBorder>
-                            <UserAvatar user={user} displayStatus={true} />
-                        </ListItemColumn>
+                value={this.state.selectedUser}
+                onChange={(user) => {
+                    this.props.onSelect(user);
+                }}
+                getLabel={(option) => option.display_name}
+                itemTemplate={
+                    (props) => {
+                        const user = props.option;
 
-                        <ListItemColumn ellipsisAndGrow>
-                            <ListItemRow>{user.display_name}</ListItemRow>
-                            <ListItemRow>@{user.username}</ListItemRow>
-                        </ListItemColumn>
-                    </ListItem>
-                )}
+                        return user == null
+                            ? (
+                                <div>
+                                    {gettext('Select a user')}
+                                </div>
+                            )
+                            : (
+                                <div style={{display: 'flex', alignItems: 'center'}}>
+                                    <UserAvatar user={user} displayStatus={true} />
+                                    <div style={{marginLeft: 14, padding: '4px 0'}}>
+                                        <div>{user.display_name}</div>
+                                        <div style={{fontSize: 12}}>@{user.username}</div>
```

Comment:
not sure why hardcoding font size? would that be accesible?

Reply:
What do you mean by hard-coding? Using pixel values or using Inline styles? Inline styles work the same as styles from the style sheet file and will be accessible.

This is a self-contained component so even if I moved them to a style sheet file, those styles wouldn't be used anywhere else, so I don't see a reason to move them.

Reply:
I mean the absolute size using pixel value

Reply:
You are right, preferred font size set in the browser is ignored when using pixels for font-sizes. I've updated it to use rem, but it isn't helping at the moment because there's a CSS rule that sets font-size to `10px` for the `html` tag, so changing that browser preference does nothing for Superdesk. Changing that 10px rule to a percent value would fix it in the sense that browser preference would be respected, but our styles [wouldn't look nice](https://user-images.githubusercontent.com/2076953/104027170-4c935d80-51c7-11eb-87df-be6d3a1674c3.png).
I'll create a separate ticket to address this.

Reply:
Added SDESK-5742